Tensile/Common.py

```diff
@@ -2207,8 +2207,9 @@ def assignGlobalParameters( config ):
     if os.name == "nt":
       globalParameters["CurrentISA"] = (9,0,6)
       printWarning("Failed to detect ISA so forcing (gfx906) on windows")
-  if globalParameters["CurrentISA"] == (9,4,2):
-    printWarning("HardwareMonitor currently disabled for gfx942")
+  if globalParameters["CurrentISA"] == (9,4,2) or globalParameters["CurrentISA"] == (11,0,0) or \
```

Comment:
Do we need this change here? How is it different from #1801?

Reply:
This is same change as 1801. if this PR CI goes fine, I will discard the 1801 PR. This PR disables ROCM SMI for gfx1100 and also take care of new PCI format changes. PR 1801 is just disable ROCm SMI for gfx1100.